ui/app/components/send/send-content/send-to-row/send-to-row.utils.js

```diff
@@ -1,27 +1,23 @@
 const CashaccountClass = require('cashaccounts')
 const cashaccount = new CashaccountClass()
 
+const SLPSDK = require('slp-sdk')
```

Comment:
I don't think we have enough space left in the FireFox build to include SLP-SDK in the frontend (ui files), unless it already exists in another place?

If not, will the existing isValidBchAddress and isValidSlpAddress methods work instead?

Reply:
those existing methods were the reason why `3` and `1` addresses were passing.

we can address the size if the next release doesn't make it into the firefox store.